doc/source/whatsnew/v0.25.0.rst

```diff
@@ -28,7 +28,38 @@ Other Enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- :meth:`Timestamp.strptime` will now rise a NotImplementedError (:issue:`21257`)
+.. _whatsnew_0250.api_breaking.deps:
+
+Dependencies have increased minimum versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Due to dropping support for Python 2.7, a number of optional dependencies have updated
+minimum versions. If installed, we now require:
+
++-----------------+-----------------+----------+
+| Package         | Minimum Version | Required |
++=================+=================+==========+
+| beautifulsoup4  | 4.4.1           |          |
++-----------------+-----------------+----------+
+| openpyxl        | 2.2.6           |          |
++-----------------+-----------------+----------+
+| pymysql         | 0.6.6           |          |
++-----------------+-----------------+----------+
+| pytz            | 2015.4          |          |
++-----------------+-----------------+----------+
+| sqlalchemy      | 1.0.8           |          |
++-----------------+-----------------+----------+
+| xlsxwriter      | 0.7.7           |          |
++-----------------+-----------------+----------+
+| xlwt            | 1.0.0           |          |
++-----------------+-----------------+----------+
+
+.. _whatsnew_0250.api.other:
+
+Other API Changes
+^^^^^^^^^^^^^^^^^
```

Comment:
This (and the above) reflects the structure of the breaking-changes-section in v.0.24.0